OPA 0.14 fails to compile on 386

# Expected Behavior
Importing the `topdown` package allows the project to compile successfully

# Actual Behavior
Compilation fails

# Steps to Reproduce the Problem

  1. Import the `topdown` package into a project
  2. set GOARCH=386
  3. go build

# Additional Info

**stack trace**
```
$ go build
# github.com/open-policy-agent/opa/topdown
C:\Users\jpree\go\pkg\mod\github.com\open-policy-agent\opa@v0.14.0\topdown\parse_bytes.go:84:5: constant 1000000000000 overflows multiplier
C:\Users\jpree\go\pkg\mod\github.com\open-policy-agent\opa@v0.14.0\topdown\parse_bytes.go:86:5: constant 1099511627776 overflows multiplier
```

**go env**
```
$ go env
set GO111MODULE=
set GOARCH=386
set GOBIN=C:\Go\bin
set GOCACHE=C:\Users\jpree\AppData\Local\go-build
set GOENV=C:\Users\jpree\AppData\Roaming\go\env
set GOEXE=.exe
set GOFLAGS=
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GONOPROXY=
set GONOSUMDB=
set GOOS=windows
set GOPATH=C:\Users\jpree\go
set GOPRIVATE=
set GOPROXY=https://proxy.golang.org,direct
set GOROOT=c:\go
set GOSUMDB=sum.golang.org
set GOTMPDIR=
set GOTOOLDIR=c:\go\pkg\tool\windows_amd64
set GCCGO=gccgo
set GO386=sse2
set AR=ar
set CC=gcc
set CXX=g++
set CGO_ENABLED=0
set GOMOD=C:\k2\conftest\go.mod
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m32 -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=C:\Users\jpree\AppData\Local\Temp\go-build389625958=/tmp/go-build -gno-record-gcc-switches
```

--- a/topdown/parse_bytes.go
+++ b/topdown/parse_bytes.go
@@ -14,18 +14,16 @@ import (
 	"github.com/open-policy-agent/opa/topdown/builtins"
 )
 
-type multiplier int
-
 const (
-	none multiplier = 1
-	kb              = 1000
-	ki              = 1024
-	mb              = kb * 1000
-	mi              = ki * 1024
-	gb              = mb * 1000
-	gi              = mi * 1024
-	tb              = gb * 1000
-	ti              = gi * 1024
+	none int64 = 1
+	kb         = 1000
+	ki         = 1024
+	mb         = kb * 1000
+	mi         = ki * 1024
+	gb         = mb * 1000
+	gi         = mi * 1024
+	tb         = gb * 1000
+	ti         = gi * 1024
 )
 
 // The rune values for 0..9 as well as the period symbol (for parsing floats)
@@ -46,7 +44,7 @@ var (
 )
 
 func builtinNumBytes(a ast.Value) (ast.Value, error) {
-	var m multiplier
+	var m int64
 
 	raw, err := builtins.StringOperand(a, 1)
 	if err != nil {
@@ -88,14 +86,14 @@ func builtinNumBytes(a ast.Value) (ast.Value, error) {
 		return nil, errUnitNotRecognized(unitStr)
 	}
 
-	num, err := strconv.Atoi(numStr)
+	num, err := strconv.ParseInt(numStr, 10, 64)
 	if err != nil {
 		return nil, errIntConv
 	}
 
-	total := num * int(m)
+	total := num * m
 
-	return builtins.IntToNumber(big.NewInt(int64(total))), nil
+	return builtins.IntToNumber(big.NewInt(total)), nil
 }
 
 // Makes the string lower case and removes spaces and quotation marks

